backend/supervisor.d/services.conf

```diff
@@ -101,16 +101,16 @@ startretries=3
 exitcodes=0,2
 stopsignal=INT
 stopwaitsecs=10
-stopasgroup=true
-killasgroup=true
+stopasgroup=false
+killasgroup=false
 serverurl=AUTO
 stdout_logfile=/dev/fd/1
 stdout_logfile_maxbytes=0
 
 [program:detection]
 command=/usr/local/bin/python -c "import %(program_name)s; %(program_name)s.run()"
-process_name=%(program_name)s
-numprocs=1
+process_name=%(program_name)s_%(process_num)s
```

Comment:
I would recommend to document this in the wiki instead of changing the default configuration